### DISCUSSION TITLE
Is there any possible timeframe for a circular pixel brush?

### DISCUSSION THREAD

**Discussion by Murnjendoof** [bitbof/klecks#79] [Q&A] [answered]
Obviously I can wait as long as is necessary, I don't want to put any pressure on the devs, but I just wanted to know when I could expect the circular pixel brush to be added. Kleki is the best browser-based art program I've been able to find, as I can't use anything outside of the browser and Kleki makes for a very easy transition from the Scratch bitmap costume editor, which was previously the only online "art" program that I had experience with.

**Comment by bitbof** [comment, marked as answer]
There is no timeframe for that feature, but I've noted your request.

Priority is currently the select tool, then zoomable previews, then overhauled layer tab. After that or somewhere inbetween probably something to prevent data loss, because browsers like Chrome recently push very aggressively to destroy data in "inactive" tabs.

Maybe it sneaks in as a feature during that, or more likely after.
